<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testTableWidget</class>
 <widget class="QWidget" name="testTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="20,20,60">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="90,10">
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="50,50">
         <item>
          <widget class="SPRHitoryGraphics" name="gHistoryIn" native="true"/>
         </item>
         <item>
          <widget class="SPRHitoryGraphics" name="gHistoryOut" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="bGetBaseSpectrum">
           <property name="text">
            <string>Get Base Spectrum</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bSeparatorOn">
           <property name="text">
            <string>Separator On</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bSeparatorOff">
           <property name="text">
            <string>Separator Off</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbThread">
           <property name="currentIndex">
            <number>4</number>
           </property>
           <item>
            <property name="text">
             <string>thread 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>thread 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>thread 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>thread 4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>All threads</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bSetSepar">
           <property name="text">
            <string>Set Separ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bStartSepar">
           <property name="text">
            <string>Start Separ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bStopSepar">
           <property name="text">
            <string>Stop Separ</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="SPRSeparateDetailsTable" name="separateDetailsTable"/>
      </item>
      <item>
       <widget class="SPRSpectrumSmallTable" name="kspectTable">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="70,30">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="workSeparTable">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>#</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Th</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>i_prg[0]</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>i_prd[1]</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>i_prd[2]</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>i_prd[3]</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>p_prg[0]</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>p_prd[1]</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>p_prd[2]</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>p_prd[3]</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>p_tk</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>p_tkH1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>p_tkH2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>p_tkH3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>w_count</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>s_rst1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>s_rst2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>s_rst3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>s_rst4</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>s_rst5</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>error</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="workGistorrammTable">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>#</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Thread</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Data</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Содержимое</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SPRSpectrumSmallTable</class>
   <extends>QTableWidget</extends>
   <header>sprspectrumsmalltable.h</header>
  </customwidget>
  <customwidget>
   <class>SPRSeparateDetailsTable</class>
   <extends>QTableView</extends>
   <header>sprseparatedetailstable.h</header>
  </customwidget>
  <customwidget>
   <class>SPRHitoryGraphics</class>
   <extends>QWidget</extends>
   <header>sprhitorygraphics.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
